tools/sosreport/tower.py

```diff
@@ -1,11 +1,9 @@
 # Copyright (c) 2016 Ansible, Inc.
 # All Rights Reserved.
 
-import sos
-from distutils.version import LooseVersion
+from sos.plugins import Plugin, RedHatPlugin, UbuntuPlugin
 
 SOSREPORT_TOWER_COMMANDS = [
-    "ansible --version",      # ansible core version
     "awx-manage --version", # tower version
```

Comment:
Are you removing `ansible --version` because ansible itself reports this somehow?